test/llvm/lifetime-intrinsics/lifetime-start-class.chpl

```diff
@@ -0,0 +1,15 @@
+// This test is used to verify that llvm.lifetime.start instrinsic is being emitted
+// for variables of a class type inside a procedure
+
+// CHECK: %[[REG:[0-9]+]] = bitcast %chpl_C_chpl_object** %{{[0-9]+}} to i8*
+// CHECK: call void @llvm.lifetime.start.p0i8(i64 8, i8* %[[REG]])
+
+class C {}
+
+proc mytest_class() {
+  var x: unmanaged C = new unmanaged C();
+  delete x;
+}
+
+mytest_class();
```

Comment:
I'm seeing this test fail with `start_test -compopts --fast lifetime-start-class.chpl`